nuitka/build/static_src/HelpersOperationBinaryAdd.c

```diff
@@ -46,8 +50,7 @@ PyObject *BINARY_OPERATION_ADD_OBJECT_INT(PyObject *operand1, PyObject *operand2
 
         PyObject *x = slot1(operand1, operand2);
 
-        if (x != Py_NotImplemented) {
-            if (unlikely(x == NULL)) {
+        if (x != Py_NotImplemented && (unlikely(x == NULL)) {
```

Comment:
This changes the meaning, the block contained more code, which is now not guarded.